addon/components/document-list.hbs

```diff
@@ -5,12 +5,17 @@
   <thead>
     <tr>
       {{#each-in @columns as |key settings|}}
+        {{! template-lint-disable require-presentational-children no-invalid-interactive }}
```

Comment:
I think `role="button"` should resolve these issues? then maybe you need to add `role="presentation"` on the child element.